examples/template/remix.config.js

```diff
@@ -2,8 +2,10 @@
  * @type {import('@remix-run/dev').AppConfig}
  */
 module.exports = {
-  appDirectory: "app",
-  browserBuildDirectory: "public/build",
-  publicPath: "/build/",
-  serverBuildDirectory: "build",
+  ignoredRouteFiles: [".*"],
+  // appDirectory: "app",
+  // assetsBuildDirectory: "public/build",
+  // serverBuildPath: "build/index.js",
+  // publicPath: "/build/",
+  // devServerPort: 8002
```

Comment:
```suggestion
```

Reply:
Yeah, that's automatically determined and can change to a random port if it's not available anyway. The only reason anyone would need to specify this is if they're running their app inside a docker container during development (which is extremely rare).

Reply:
PR incoming then 😉